ajax/libs/jsnes/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jsnes",
+  "description": "A JavaScript NES emulator",
+  "author": "Ben Firshman <ben@firshman.co.uk>",
+  "filename": "src/index.min.js",
```

Comment:
This point to main file under `basePath` in auto-update config, which would be `jsnes.min.js`.